packages/@aws-cdk/aws-redshift/README.md

```diff
@@ -75,4 +75,76 @@ cluster.addRotationMultiUser('MyUser', {
 });
 ```
 
-This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aws-cdk) project.
+## Database Resources
+
+This module allows for the creation of non-CloudFormation database resources via [custom
+resources](https://docs.aws.amazon.com/cdk/api/latest/docs/custom-resources-readme.html).
+These resources include database users and database tables.
+
+### Creating Users
+
+Create a user within a Redshift cluster database by instantiating a `User` construct. This
+will generate a username and password, store the credentials in a [Secrets Manager
```

Comment:
Should we consider allowing setting the username? 

Also can you include a brief description about how the username and password will be generated?

Reply:
added descriptions, including the already-existing but undocumented ability to provide a username

do you think I should add the ability to provide a password in this PR?